odoo/conf/deprecation.py

```diff
@@ -27,7 +27,7 @@
 # - The register attribute in openerp.report.interface.report_int (and in its
 # - auto column in ir.actions.report.xml.
 # inheriting classes).
-allow_local_service = True
+allow_local_service = False
```

Comment:
Is there a point in even leaving this thing?

Also allow_report_int_registration below since the corresponding report/service was also removed.